src/components/Info/Information.styles.ts

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import resetCSS from "../../styles/reset";
+import fonts from "../../styles/fonts";
+import colors from "../../styles/colors";
+
+export const CardStyled = styled.div`
+  ${resetCSS}
+  ${fonts.text}
+  ${fonts.textBold}
+  border: 2px solid #E6EDFF;
```

Comment:
go for blueSkyLight i think the designer is going a bit mad with these blues, i'll chat to him later

Reply:
done